.github/workflows/ci.yaml

```
@@ -303,8 +303,14 @@ jobs:
           Export-Certificate @params
           certutil -encode $home/tmp-cert.cer $home/cosmosdbcert.cer
           Remove-Item $home/tmp-cert.cer
-          & ${env:JAVA_HOME}/bin/keytool.exe -keystore ${env:JAVA_HOME}/jre/lib/security/cacerts -storepass 'changeit' -importcert -noprompt -alias cosmos_emulator -file $home/cosmosdbcert.cer
-          & ${env:JAVA_HOME}/bin/keytool.exe -keystore ${env:JAVA_HOME}/jre/lib/security/cacerts -storepass 'changeit' -list -alias cosmos_emulator
+          # Setting the keystore option differs between Java 8 and Java 11+
+          if ( ${env:INT_TEST_JAVA_RUNTIME_VERSION} -eq '8' ) {
+            $keystore = "-keystore", "${env:JAVA_HOME}/jre/lib/security/cacerts"
```

```suggestion
            $keystore = "-keystore ${env:JAVA_HOME}/jre/lib/security/cacerts"
```
Writing the variable like this will not expand correctly in Powershell because it does not recognize `-keystore` as being an option for the `keytool.exe` command. 
Instead, we need to separate the option from its value by storing it as an array in the `$keystore` variable.